scripts/docker/hub/Dockerfile

```diff
@@ -10,27 +10,23 @@ RUN apt update && apt install -y --no-install-recommends openssl libudev-dev fil
 # show backtraces
 ENV RUST_BACKTRACE 1
 
-#cleanup Docker image
-RUN apt autoremove -y
-RUN apt clean -y
-RUN rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
+# cleanup Docker image
+RUN apt autoremove -y \
+  && apt clean -y \
+  && rm -rf /tmp/* /var/tmp/* /var/lib/apt/lists/*
 
 RUN groupadd -g 1000 parity \
   && useradd -m -u 1000 -g parity -s /bin/sh parity
 
-
-#add TARGET to docker image
+# add TARGET to docker image
 COPY artifacts/x86_64-unknown-linux-gnu/$TARGET /bin/$TARGET
```

Comment:
TARGET should probably then also removed here and made more clear/changed in the publish-docker.sh

Reply:
Well the variable doesn't make sense anymore when the flexibility of the container executable is removed i.e.
https://github.com/paritytech/parity-ethereum/blob/master/.gitlab-ci.yml#L96
and https://github.com/paritytech/parity-ethereum/blob/master/scripts/gitlab/publish-docker.sh#L16 ff
Unfortunately docker doesn't allow variables in the entrypoint.